device/mellanox/x86_64-mlnx_msn3800-r0/Mellanox-SN3800-D24C52/port_config.ini

```diff
@@ -0,0 +1,77 @@
+# name       lanes            alias      index    speed
```

Comment:
@madhanmellanox Please use the same alignment as in this PR: https://github.com/Azure/sonic-buildimage/pull/4085

Reply:
@@madhanmellanox do we need fec for this SKU?

Reply:
The SKU is generated by sonic_sku_create.py and it is using tabulate method of tabulate module, that is the reason for the alignment.
FEC is not present in the 2700 SKUs. So, FEC is not mandatory.